src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/pages/CropPage.kt

```diff
@@ -76,7 +69,7 @@ class CropPage(val crop0: () -> CropType, sizeX: Int, sizeY: Int, paddingX: Int
                 2,
                 verticalAlign = RenderUtils.VerticalAlignment.BOTTOM
             ),
-            144,
+            50,
```

Comment:
```suggestion
            164,
```

Reply:
lgtm 👍
![2024-06-11_12 27 47](https://github.com/hannibal002/SkyHanni/assets/88066006/ef01eab2-1cd1-4faa-a1b1-eb4f44f4ed95)

Reply:
not ideal, but it requires more effort because some of it's hardcoded and will be worked on at a later point in time